tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -55,40 +55,36 @@ <h3 class="section-title">General</h3>
   <div
     class="control-row scalars-step-selector"
     *ngIf="isScalarStepSelectorFeatureEnabled"
+    [title]="!isAxisTypeStep()? 'Only available when Horizontal Axis is set to step': ''"
   >
     <mat-checkbox
       [checked]="isScalarStepSelectorEnabled"
       (change)="stepSelectorToggled.emit()"
       [disabled]="!isAxisTypeStep()"
-      [title]="!isAxisTypeStep()? 'Only available When Horizontal Axis is set to step': ''"
       >Enable step selection and data table
     </mat-checkbox>
     <span class="indent">(Scalars only)</span>
-  </div>
-
-  <div
-    class="control-row linked-time"
-    *ngIf="isLinkedTimeFeatureEnabled && xAxisType == XAxisType.STEP"
-  >
-    <label>Link visualization by step</label>
-    <div class="controls">
-      <div>
-        <mat-checkbox
-          [checked]="isLinkedTimeEnabled"
-          (change)="linkedTimeToggled.emit()"
-          >Enabled</mat-checkbox
-        >
-      </div>
-      <div class="step-selector">
-        <tb-range-input
-          [min]="stepMinMax.min"
-          [max]="stepMinMax.max"
-          [lowerValue]="linkedTimeSelection?.start.step"
-          [upperValue]="linkedTimeSelection?.end?.step"
-          [enabled]="isLinkedTimeEnabled"
-          (singleValueChanged)="onSingleValueChanged($event)"
-          (rangeValuesChanged)="onRangeValuesChanged($event)"
-        ></tb-range-input>
+    <div class="control-row indent" *ngIf="isLinkedTimeFeatureEnabled">
+      <div class="linked-time">
+        <div>
```

Comment:
Is this div necessary?

Reply:
Thanks James! This is indeed an unnecessary layer. Removed it.